wiki/en/en-Walk-Through.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: wiki
+title: "Jamulus Walk-Through"
+lang: "en"
+permalink: "/wiki/Walk-Through"
+---
+# Jamulus walk-through
+Let's have a look at what Jamulus looks like and what you should do after having installed the software.
+## First start
+Before you start Jamulus:
+1. Close all other applications
```

Comment:
I wonder if we should say "For now, close all other applications" since you may be able to run multiple things on the machine, but it's best to start from zero at first.

Reply:
Changed it and added "It's best to start from zero at first"